src/ibusenginesimple.c

```diff
@@ -428,6 +428,60 @@ ibus_engine_simple_commit_str (IBusEngineSimple *simple,
     g_free (backup_str);
 }
 
+static gboolean
+check_compose_sequence (guint     *keys,
```

Comment:
I think `ibus_engine_simple_check_all_compose_table()` should use `check_compose_sequence()`.